packages/docs/cookbook/testing.md

```diff
@@ -94,23 +94,54 @@ store.name = 'my new name'
 store.$patch({ name: 'new name' })
 expect(store.name).toBe('new name')
 
-// actions are stubbed by default but can be configured by
-// passing an option to `createTestingPinia()`
+// actions are stubbed by default, so this call won't actually
+// execute the implementation defined by the store. See below 
+// to customize this behavior.
 store.someAction()
 
 expect(store.someAction).toHaveBeenCalledTimes(1)
 expect(store.someAction).toHaveBeenLastCalledWith()
 ```
 
-If you are not using Jest (for example, you are using vitest), you will need to provide a [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.testingoptions.html#createspy) to `createTestingPinia()`, like:
+Please note that if you are using Vue 2, `@vue/test-utils` requires a [slightly different configuration](#unit-test-components-vue-2).
+
+
+### Customizing behavior of actions
+
+By default, `createTestingPinia` stubs out all store actions, such that they are not executed when called during unit tests. This allows you to test your components and stores separately.
+
+If you want your actions to execute during unit tests, specify `stubActions: true` as an option when calling `createTestingPinia`:
 
 ```
-import { fn } from 'vitest'
-//...
-plugins: [createTestingPinia({ createSpy: fn })]
+const wrapper = mount(Counter, {
+  global: {
+    plugins: [createTestingPinia({ stubActions: false })],
+  },
+})
+
+const store = useSomeStore()
+
+// Now this call WILL execute the implementation defined by the store
+store.someAction()
+
+// ...but it's still wrapped with a spy, so you can inspect calls 
+expect(store.someAction).toHaveBeenCalledTimes(1)
+
 ```
 
-Please note that if you are using Vue 2, `@vue/test-utils` requires a [slightly different configuration](#unit-test-components-vue-2).
+### Specifying the createSpy function
+
+When using Jest, or vitest with `globals: true`, `createTestingPinia` will create action stubs automatically using the spy function defined by the test framework (`jest.fn` or `vitest.fn`). If you are using a different framework, you'll need to provide a [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.testingoptions.html#createspy) and supply it as a parameter.
+
+```
+import sinon from 'sinon'
+createTestingPinia({ 
+  createSpy: sinon.spy // use sinon's spy to wrap actions
+  })
+```
```

Comment:
```suggestion
```js
import sinon from 'sinon'

createTestingPinia({ 
  createSpy: sinon.spy // use sinon's spy to wrap actions
})
```
```